xml/System.IO.Pipelines/FlushResult.xml

```diff
@@ -39,9 +39,9 @@
         <Parameter Name="isCompleted" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="isCanceled">To be added.</param>
-        <param name="isCompleted">To be added.</param>
-        <summary>Creates a new instance of <see cref="T:System.IO.Pipelines.FlushResult" /> setting <see cref="P:System.IO.Pipelines.FlushResult.IsCanceled" /> and <see cref="P:System.IO.Pipelines.FlushResult.IsCompleted" /> flags.</summary>
+        <param name="isCanceled"><see langword="true" /> to indicate the current <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> operation that produced this <see cref="T:System.IO.Pipelines.FlushResult" /> was canceled by <see cref="M:System.IO.Pipelines.PipeWriter.CancelPendingFlush()" />; otherwise, <see langword="false" />.</param>
+        <param name="isCompleted"><see langword="true" /> to indicate the current <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> operation was completed; otherwise, <see langword="false" />.</param>
```

Comment:
FlushResult.IsCompleted is similar to ReadResult.IsCompleted

FlushResult.IsCompleted being set means that there is no one is reading or will be reading the data being written. Basically it just means all the writes/flushes are being ignored, so the writer can (but is not required to) stop writing for efficiency's sake. In the case of a PipeWriter/PipeReader pair, this means that PipeReader.Complete() was called. A PipeWriter that simply wraps a stream will never return a FlushResult with the IsCompleted flag set, since there's no way to know when the reading side has completed.

```suggestion
        <param name="isCompleted"><see langword="true" /> to indicate there is no longer anything reading the data that's being written to the <see cref="T:System.IO.Pipelines.PipeWriter" />.</param>
```

This is somewhat analogous to the comment I made about ReadResult.IsCompleted: https://github.com/dotnet/dotnet-api-docs/pull/3491/files#r346607748

Reply:
Possible rephrase? A little more concise. I felt "anything" was a little vague here. A "thing reading the data" is, by definition, "a reader" ;).

```suggestion
        <param name="isCompleted"><see langword="true" /> to indicate the reader is no longer reading data written to the <see cref="T:System.IO.Pipelines.PipeWriter" />.</param>
```